test/lambda/graphql.test.js

```diff
@@ -32,9 +32,12 @@ test.before(async () => {
 test.after.always(async () => {
   delete process.env.COMPOSE_PROJECT_NAME;
   try {
-    container.stop();
+    if (container) {
+      await container.stop();
+      console.log(`Stopped container ${container.id}`);
```

Comment:
Should be a debug statement?

Reply:
This is also a test file, so we aren't using the debug library.